core/client/app/templates/components/gh-profile-image.hbs

```diff
@@ -2,7 +2,7 @@
     {{#unless hasUploadedImage}}
     <div class="placeholder-img" style={{defaultImage}}></div>
```

Comment:
Can you indent template blocks, please?  It's really hard to read when everything is at a single level.